### PR TITLE
High score

### PR DESCRIPTION
**Pull request by ilhs1041495**
High score is basically done, might need some clean up though.

### CODE REVIEW

**Review by roguefirework — DISMISSED** [review]
Please add some way to get the top ten

**Review by MaximumMaxxx — APPROVED** [review]
I approve

**Comment by ilhs1041495** [comment]
🚟

**Comment by OdinDaDoge** [comment]
🚟

**Comment by ilhs1041495** [comment]
> 🚟
🚆

File: Core/WumpusTesting/HighScoreTests.cs
Comment (OdinDaDoge) [inline Core/WumpusTesting/HighScoreTests.cs:1]: What does this file do?

**Review by OdinDaDoge — APPROVED** [review]
Files and Code looks good and fairly clean, should work with testing as long as the code properly builds.

**Pull request merged**